aks/main.tf

```diff
@@ -13,5 +13,9 @@ module "loki-stack" {
   ingress          = var.ingress
   enable_filebeat  = var.enable_filebeat
 
+  sensitive_values = merge({
+    "loki-distributed.loki.storageConfig.azure.account_key" = var.logs_storage.storage_account_key
```

Comment:
Correct me if I'm wrong, but this config will populate the structure:
```yaml
loki-distributed:
  loki:
    storageConfig:
      azure:
        account_key: blablabla
```

But what about the need to pass the same value to the non-distributed Loki on line 20 that should reproduce the structure:

```yaml
loki-stack:
  loki:
    config:
      storage_config:
        azure:
          account_key: blablabla
```

Hope you understood what I was talking about 😉

Reply:
Good point, I'll add support for loki-stack.